src/app/components/EmbedConsentBanner/ConsentBanner.styles.tsx

```diff
@@ -0,0 +1,58 @@
+import { css, Theme } from '@emotion/react';
+
+export default {
+  parent: ({ spacings, palette }: Theme) =>
+    css({
+      backgroundColor: palette.WHITE,
+      padding: `${spacings.QUADRUPLE}rem ${spacings.DOUBLE}rem`,
+      minHeight: 280,
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'center',
+      alignItems: 'flex-start',
+    }),
+
+  heading: ({ palette }: Theme) =>
+    css({
+      color: palette.GREY_10,
+    }),
+
+  textBody: ({ spacings, palette }: Theme) =>
+    css({
+      color: palette.GREY_10,
+      margin: `${spacings.DOUBLE}rem 0`,
+
+      a: {
+        color: 'inherit',
+        textDecoration: 'none',
+        borderBottom: `1px solid ${palette.GREY_10}`,
```

Comment:
@amoore108 we usually use rem even for borders (same for hover, focus and visisted)